src/test/java/com/kainos/ea/backend/services/JobFamilyServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package com.kainos.ea.backend.services;
+
+import com.kainos.ea.backend.models.JobFamily;
+import com.kainos.ea.backend.repositories.JobFamilyRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class JobFamilyServiceTest {
+    @Mock
+    private JobFamilyRepository jobFamilyRepository;
+
+    @Test
+    void when_getJobFamiliesByCapabilityName_expectRepositoryCalledPassback() {
```

Comment:
Could you add an underscore after `expect` so it is consistent with our tests?
`when_getJobFamiliesByCapabilityName_expect_RepositoryCalledPassback()`